pandas/tests/indexing/test_loc.py

```diff
@@ -1163,6 +1163,25 @@ def test_loc_getitem_listlike_all_retains_sparse(self):
         result = df.loc[[0, 1]]
         tm.assert_frame_equal(result, df)
 
+    @td.skip_if_no_scipy
+    def test_loc_sparse_frame(self):
```

Comment:
can you start the name with `test_loc_getitem` to match the pattern in this clas

mainly this is testing that .loc indexing retains SparseDtype right?  should test for Series too?

Reply:
Have updated the name, and also added similar tests for sparse series, or would it be better to brake these out into a seperate function?

Reply:
yeah i think if you can move the series tests to series/indexing/test_indexing

Reply:
I have moved them to that folder.

Reply:
sorry fellas, rule of thumb: if you're testing .loc, it belongs in tests.indexing.test_loc.  See https://github.com/pandas-dev/pandas/blob/master/doc/source/development/test_writing.rst

Reply:
No problem, have now moved it back to `tests.indexing.test_loc`.